create_a_materialized_view_batch_ingest/.tinyenv

```diff
@@ -1 +1 @@
-VERSION=0.0.0
+VERSION=0.0.1
```

Comment:
Version 0.0.1 auto-promotes the release, so the changes will be deploy to "live". If a backfill is required you need to leave your changes in preview, for that bump the major version.